index.html

```diff
@@ -71,12 +71,39 @@ <h2>I am a Software Developer</h2>
 		<div class="text1">
 			<h2>My Recent Works <img src="images/line.png" alt=""></h2>
 		</div>
+
+		<div class="popupcontainer" id="popup">
+			<div class="popupcontainer2">
+				<div class="popuptitle">
+					<h1 class="header2">Multi Post Story</h1>
+					<button class="closebtn active" type="button" onclick="togglePopup()">&times;</button>
+				</div>
+				<div class="langcontainer">
+					<ul class="ulcontainer">
+						<li class="langg">html</li>
+						<li class="langg">bootstrap</li>
+						<li class="langg">Ruby on rails</li>
+					</ul>
+				</div>
+				<div class="popupcontainer3">
+					<img src="images/Snapshoot Portfolio.png" class="image2" alt="">
+					<div class="description1">
+						<p class="paragraph2">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+						<div class="btncontainer">
+							<button class="popbtn" type="button"><a href="https://mikey1-nuel.github.io/Setup-and-Mobile-First/"><img src="images/Enable.png" alt="enable"></a></button>
+							<button class="popbtn" type="button"><a href="https://github.com/Mikey1-nuel/Setup-and-Mobile-First"><img src="images/Enable2.png" alt=""></a></button>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225255976-b07492ad-ccf2-429d-b69a-a7348540f33b.png)

- [ ] it would be nice if you remove this section description to make your readme file descriptive and clean 😊 This is just an instruction for you for the very first time when you add this readme file 👍 After adding this file these section descriptions and all other instructions should be removed from the readme file 👍